include/picolibrary/indicator.h

```diff
@@ -28,6 +28,14 @@
  */
 namespace picolibrary::Indicator {
 
+/**
+ * \brief Initial indicator state.
+ */
+enum class Initial_Indicator_State {
+    ILLUMINATED,
+    EXTINGUISHED,
```

Comment:
Please add inline comments (`///< Foo.`) for each enumerator for consistency with other library enums.

The following suggestion may not pass the formatting check.
```suggestion
    ILLUMINATED, ///< Illuminated.
    EXTINGUISHED, ///< Extinguished.
```